src/L2ERC20.sol

```diff
@@ -25,4 +32,16 @@ contract L2ERC20 is ERC20Votes, WormholeReceiver {
     (address account, uint256 amount) = abi.decode(vm.payload, (address, uint256));
     _mint(account, amount);
   }
+
+  /// @dev Clock used for flagging checkpoints.
+  function clock() public view override returns (uint48) {
+    return SafeCast.toUint48(l1Block.number());
+  }
+
+  /// @dev Description of the clock
+  function CLOCK_MODE() public view virtual override returns (string memory) {
+    // Check that the clock was not modified
+    require(clock() == l1Block.number(), "ERC20Votes: broken clock mode");
+    return "mode=l1blocknumber&from=default";
```

Comment:
Is this mode correct?

Reply:
Ah thank you! That makes sense

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/9/commits/fd42a3419059f339649f96482b20c2f9278384f3